app/experimenter/kinto/README.md

```diff
@@ -0,0 +1,158 @@
+# Remote Settings (Kinto) Integration
+
+# Table of Contents
```

Comment:
Q: Should this be an H1 or H2?

Reply:
Yeah, this should probably be changed to `##`